demo/kubernetes/rook-cluster.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   versionTag: master-latest
   dataDirHostPath:
+  placement:
```

Comment:
Added an example for 'all' and the structure for the rest, let me know if it looks okay.